data/yam/agama/auto/lib/storage.libsonnet

```diff
@@ -68,10 +68,62 @@ local whole_disk_and_boot_unattended() = {
     device: 'boot-disk',
   },
 };
+
+local raid() = {
+  drives: [
+    {
+      partitions: [
+        {
+          search: '*',
+          delete: true
+        },
+        {
+          id: 'bios_boot',
+          size: '8 MiB'
+        },
+        {
+          alias: 'first-raid',
+          id: 'raid',
+          size: '7.81 GiB'
+        },
+        {
+          alias: 'second-raid',
+          id: 'raid',
+          size: '512 MiB'
+        },
+      ],
+      mdRaids: [
+        {
+          level: 'raid0',
```

Comment:
we should be able to pass `0` here somehow.

Reply:
updated it, thanks.